doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -4,6 +4,8 @@
 
 SSL_SESSION_get_time, SSL_SESSION_set_time, SSL_SESSION_get_timeout, SSL_SESSION_set_timeout - retrieve and manipulate session time and timeout settings
 
+SSL_SESSION_get0_hostname - retrieve the hostname for the session
```

Comment:
The perl script is called "util/extract-names.pl", but it is never run manually. It gets triggered during installation of the documentation automatically. If you run "make install_docs" and then "man SSL_SESSION_get0_hostname" you should see the page relevant to your function. If it works everything was ok.